gaius_common/middleware/changeLog.py

```diff
@@ -0,0 +1,46 @@
+import uuid
+import requests
+from contextvars import ContextVar
+
+_request_ctx = ContextVar('current_request', default={})
+
+def set_current_request(new_context):
+    _request_ctx.set(new_context)
+
+def get_current_request():
+    return _request_ctx.get()
+
+def add_to_request_context(key, value):
+    context = _request_ctx.get()
+    context[key] = value
+    _request_ctx.set(context)
+
+def get_ip_address(request):
+    x_forwarded_for = request.META.get('HTTP_X_FORWARDED_FOR')
```

Comment:
add x-real-ip also at the first

Reply:
This is fixed. I've added x-real-ip at the first.